src/Akeneo/Category/front/src/feature/pages/TemplateIndex.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {FC, useEffect, useState} from 'react';
+import {Breadcrumb, SkeletonPlaceholder, TabBar, useBooleanState, useTabBar} from 'akeneo-design-system';
+import {
+  FullScreenError,
+  PageContent,
+  PageHeader,
+  PimView,
+  useRouter,
+  useSecurity,
+  useSessionStorageState,
+  useTranslate,
+} from '@akeneo-pim-community/shared';
+import {useCategoryTree} from '../hooks';
+import {useParams} from 'react-router';
+
+const attributeTabName = '#pim_enrich-template-tab-attribute';
+const propertyTabName = '#pim_enrich-template-tab-property';
+
+type Params = {
+  treeId: string;
+  templateId: string;
+};
+
+const TemplateIndex: FC = () => {
+  const {treeId, templateId} = useParams<Params>();
+  const router = useRouter();
+  const translate = useTranslate();
+  const [isModalOpen, openModal, closeModal] = useBooleanState();
+  const {isGranted} = useSecurity();
+
+  const {tree, loadingStatus, loadTree} = useCategoryTree(parseInt(treeId), '-1');
+  const templateLabel = 'Template Label';
+
+  const [treeLabel, setTreeLabel] = useState<string>('');
+  const followSettingsIndex = () => router.redirect(router.generate('pim_settings_index'));
+  const followCategoriesIndex = () => router.redirect(router.generate('pim_enrich_categorytree_index'));
+  const followCategoryTree = () => {
+    if (!tree) {
+      return;
+    }
+    router.redirect(router.generate('pim_enrich_categorytree_tree', {id: tree.id}));
+  };
+
+  const [activeTab, setActiveTab] = useSessionStorageState(attributeTabName, 'pim_category_activeTab');
+  const [isCurrent, switchTo] = useTabBar(activeTab);
+
+  useEffect(() => {
+    loadTree();
+  }, [loadTree, treeId]);
```

Comment:
We should remove the keys on this hook to avoid unexpected calls to the backend. Running this effect only when the page is mounted should be enough.

```suggestion
  useEffect(() => {
    loadTree();
  }, []);
```